internal/mk/types.go

```diff
@@ -0,0 +1,7 @@
+package mk
+
+import "github.com/hazelcast/hazelcast-commandline-client/internal"
+
+func ValueFromString(value, valueType string) (any, error) {
+	return internal.ConvertString(value, valueType)
```

Comment:
Why not use `internal.CovertString()` directly? 🤔

Reply:
I'll probably remove that function.

Reply:
Clarification: I'll probably remove `internal.ConvertString` in the future.